### PR TITLE
[TASK] Advanced: Add defaultripple flag option on settings (RT-3264)

### PR DESCRIPTION
**Pull request by yongsoo**

### CODE REVIEW

**Comment by mrajvanshy** [comment]
@yongsoo needs to be rebased

**Comment by yongsoo** [comment]
@mrajvanshy @vhpoet please merge this asap after travis build passes, if it LGTY.  Thank you.

File: src/js/tabs/advanced.controller.js [hunk 1]
```diff
@@ -77,6 +78,21 @@ AdvancedTab.prototype.angular = function(module)
...
+
+          keychain.requestSecret(id.account, id.username, function (err, secret) {
+            if (err) {
+              console.log('Error: ', err);
+              return;
+            }
+            tx.secret(secret);
+            tx.submit();
+          });
+
```
Comment (rht) [inline src/js/tabs/advanced.controller.js]: Put ```break``` here.

Comment (yongsoo) [inline src/js/tabs/advanced.controller.js]: Thank you, I forgot it. I will submit another PR tomorrow.

**Comment by vhpoet** [comment]
LGTM

**Pull request merged**